src/brevitas/graph/equalize.py

```diff
@@ -166,8 +225,16 @@ def _equalize(model, regions, iterations):
         if name in name_set:
             name_to_module[name] = module
     for i in range(iterations):
+        print(f"Equalize iter {i}")
+        scale_factor_max = torch.tensor(0.0)
```

Comment:
Same issue as before when it comes to torch.tensor. I would initialize it to None, and the allocate a tensor later on, after you compute scale_factor_region_max,  with the same device and dtype of scale_factor_region_max, checking if scale_factor_max is None